scripts/notify-suppliers-of-new-questions-answers.py

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env python
+"""
+Send email notifications to supplier users when a question has been answered
+
+Usage: notify-suppliers-of-new-questions-answers.py <stage> <api_token> <mandrill_api_key> [options]
+
+Options:
+    --dry-run                               List notifications that would be sent without sending the emails
+    --exclude-supplier-ids=EXCLUDE-SUPPLIERS   Comma separated list of suppliers IDs to be excluded. This is incase the
```

Comment:
Also is a comma-separated list the best thing?  What if there are 100s of suppliers who already received the thing and only a couple left to go?  Maybe better to read from a file? (Although, if running on Jenkins I guess that is more difficult..)